beta/kustomize/README.md

```diff
@@ -0,0 +1,196 @@
+# Deploy 1Password SCIM Bridge on Kubernetes with Kustomize
+
+*Learn how to deploy 1Password SCIM Bridge on a Kubernetes cluster using Kustomize.*
+
+> **Note**
+>
+> If you use Azure Kubernetes Service, learn how to [deploy the SCIM bridge there](https://support.1password.com/scim-deploy-azure/).
+
+**Table of contents:**
+
+- [Before you begin](#before-you-begin)
+- [Step 1: Deploy 1Password SCIM Bridge to the Kubernetes cluster](#step-2-deploy-1password-scim-bridge-to-the-kubernetes-cluster)
+- [Step 2: Create a DNS record](#step-3-create-a-dns-record)
+- [Step 3: Configure Let's Encrypt](#step-4-configure-lets-encrypt)
+- [Step 4: Test the SCIM bridge](#step-5-test-the-scim-bridge)
+- [Step 5: Connect your identity provider](#step-6-connect-your-identity-provider)
+- [Update your SCIM Bridge](#update-your-scim-bridge)
+- [Appendix: Resource recommendations](#appendix-resource-recommendations)
+- [Appendix: Customize your deployment](#appendix-customize-your-deployment)
+
+## Before you begin
+
+Before you begin, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there.
+
+### In this folder
+
+- [`/base`](./base/): The default deployment that the overlays build on top of to
+deploy your preferred bridge.
+- [`/overlays`](./overlays/): The overlays to customize your deployment.
+
+## Step 1: Prepare your `scimsession` file
+
+After downloading your `scimsession` file after creating your SCIM bridge credentials,
+place it in the folder `./base`. The path should be `./base/scimsession`. Kubernetes
+will use this file along with a secret generator to create the Kubernetes secret
+containing the contents of your `scimsession` file, named as `scimsession`.
+
+## Step 2: Determine which overlays to use
+
+There are currently 3 overlays defined for changing  the bridge as well as a deployment
+overlay. In most standard deployments, the `letsencrypt` overlay is going to be used.
+This overlay configures the SCIM bridge to request a certificate from Let's Encrypt using
+a domain name defined in the overlay ConfigMap. If you're bringing your own 
+TLS certificate, the `self-managed-tls` overlay is to be used instead.
+
+If you're using Google Workspace, you'll also need the `workspace` overlay configured
+as documented in that overlay's README. 
+
+## Step 3: Configure the overlays
+
+### Let's Encrypt
+
+To configure the Let's Encrypt overlay, edit the file `patch-configmap.yaml`. Update
```

Comment:
I wonder if the phrasing here might be confusing since at this point, someone may not have created the A record since they'd not have deployed anything to the cluster and may not have an external IP?